src/app/scopes-dialog/scopes.ts

```diff
@@ -651,5 +651,21 @@ export const PermissionScopes: IPermissionScope[] = [
       'Allows the app to deliver its notifications, on your behalf. Also allows the app to read, update, and delete your notification items for this app',
     preview: false,
     admin: false
+  },
+  {
+    name: 'PrivilegedAccess.ReadWrite.AzureResources',
```

Comment:
This scope name doesn't match the docs.  Which is correct? https://docs.microsoft.com/en-us/graph/permissions-reference

Reply:
@darrelmiller The names in my PR are the correct ones. Can we update the docs you pointed accordingly as well?

Reply:
@wn9237 Updating the permissions doc above is your responsibility Nan.  Please go ahead and create a PR to fix this.